website/src/pages/showcase/index.js

```diff
@@ -0,0 +1,81 @@
+const React = require("react");
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import './styles.scss';
+
+const Container = (props) => {
+  return (
+      <div className="hero text--center showcase">
+        <div className="container">
+          {props.children}
+        </div>
+      </div>
+  );
+}
+
+const Header = (props) => {
+  return (
+      <div className="header">
+        <div className="title">
+          Use cases
+        </div>
+        <div className="tips">
+          This project is used by all these folks
+          <br />
+          Are you using this project?&nbsp;
+          <a
+              href="https://github.com/apache/apisix/blob/master/powered-by.md"
+              target="_blank"
+              rel="noopener"
```

Comment:
```suggestion
              rel="noopener external nofollow"
```
would be better to focus on our own flow and not brain it to the outside